recipes/centos_spring_boot/Dockerfile

```diff
@@ -16,7 +16,12 @@ ARG JUNIT_VERSION=4.12
 ENV SPRING_BOOT_GROUP=org.springframework.boot
 
 COPY install_spring_boot_dependencies.sh /tmp/
+COPY ./run.sh ${HOME}/
 RUN sudo chown user:user /tmp/install_spring_boot_dependencies.sh && \
     sudo chmod a+x /tmp/install_spring_boot_dependencies.sh && \
     scl enable rh-maven33 /tmp/install_spring_boot_dependencies.sh && \
-    sudo rm -f /tmp/install_spring_boot_dependencies.sh
+    sudo rm -f /tmp/install_spring_boot_dependencies.sh && \
+    sudo chown user:user ${HOME}/run.sh && \
+    sudo chmod +x ${HOME}/run.sh
```

Comment:
we could merge the two chown and chmod instructions as we can do `chmod <options> file1 file2`

Reply:
sorry @benoitf - i never used that command :(